src/coreclr/jit/importer.cpp

```diff
@@ -4097,7 +4097,8 @@ GenTree* Compiler::impImportCnsTreeFromBuffer(uint8_t* buffer, var_types valueTy
 GenTree* Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolvedToken,
                                               CORINFO_ACCESS_FLAGS    access,
                                               CORINFO_FIELD_INFO*     pFieldInfo,
-                                              var_types               lclTyp)
+                                              var_types               lclTyp,
+                                              /* OUT */ bool*         pIsHoistable)
```

Comment:
Add a header comment? I know we still have many without these comments but I try to add one for the methods I'm changing.

Reply:
I attempted one.. learning a bit about those types in the process.

(the merge conflict was between code that I moved and main removed)